ext/skeleton/config.m4.in

```diff
@@ -22,7 +22,6 @@ if test "$PHP_%EXTNAMECAPS%" != "no"; then
   dnl Write more examples of tests here...
 
   dnl # get library FOO build options from pkg-config output
-  dnl AC_PATH_PROG(PKG_CONFIG, pkg-config, no)
```

Comment:
We should update the skeleton to use `PKG_CHECK_MODULES`, so new extensions do this right :) Not necessary as part of this PR...

Reply:
Sure :D